### DISCUSSION TITLE
Discount Limits

### DISCUSSION THREAD

**Discussion by sbts** [ledgersmb/LedgerSMB#5849] [Ideas]
All items should have the ability to have a minimum and maximum price set.
It should be possible to set this as a percentage, delta, or absolute value.

One way to manage the type of limit would be
- zero value is no limit
- -1> x <1 is a percentage
- 1st char in [-+] is a relative adjustment
- non prefixed number is an absolute value

**Comment by ehuelsmann** [comment]
I understand the desire, but feedback has been for years that we made things too complex or rather insufficiently self explanatory. How do we prevent this proposal from adding to that sentiment?

**Comment by sbts** [comment]
Onscreen help would be my response. 
Some fixed help text possibly with a tooltip or help button alongside the field.

That's how other accounting and POS software I've seen handles help on fields like this

**Comment by ehuelsmann** [comment]
My point isn't the entry of this field, but the fact that we're introducing yet another configuration field which is likely to increase the sense of complexity.
